.github/workflows/release-publish.yml

```diff
@@ -55,12 +55,12 @@ jobs:
         continue-on-error: true
         run: curl -u docsWorkflows:${{ secrets.N8N_WEBHOOK_DOCS_PASSWORD }} --request GET 'https://internal.users.n8n.cloud/webhook/trigger-release-note' --header 'Content-Type:application/json' --data '{"version":"${{env.RELEASE}}"}'
 
-      - name: Merge Release into 'master'
-        run: |
-          git fetch origin
-          git checkout --track origin/master
-          git config user.name "Jan Oberhauser"
-          git config user.email jan.oberhauser@gmail.com
-          git merge --ff n8n@${{env.RELEASE}}
-          git push origin master
-          git push origin :${{github.event.pull_request.base.ref}}
+      # - name: Merge Release into 'master'
```

Comment:
Why is it necessary to remove this part of the code?

Reply:
this has never worked. commenting this out until I can figure out how o make this work. 
I could instead delete this, and try to do this step in a n8n workflow instead.